lib/public/views/Logs/Create/LogCreationModel.js

```diff
@@ -85,9 +86,10 @@ export class LogCreationModel extends Observable {
         this._createdLog = RemoteData.loading();
         this.notify();
 
-        const { title, text, parentLogId, runNumbers, environments, attachments } = this;
+        const { title, text, parentLogId, runNumbers, environments, lhcFills, attachments } = this;
         const tagsTexts = this.tagsPickerModel.selected;
         const environmentsIds = environments ? environments.split(',').map((x) => x.trim()) : [];
+        const lhcFillNumbers = lhcFills ? lhcFills.split(',').map((x) => x.trim()) : [];
```

Comment:
I have not noticed, but in general please always use useful variable, even in map, for loop and so on. Here why not lhcFillNumber?